opencti-platform/opencti-graphql/src/domain/stixCoreObject.js

```diff
@@ -406,7 +406,7 @@ export const stixCoreObjectImportPush = async (context, user, id, file, args = {
       const key = `${filePath}/${filename}`;
       meta.external_reference_id = generateStandardId(ENTITY_TYPE_EXTERNAL_REFERENCE, { url: `/storage/get/${key}` });
     }
-    const { upload: up, untouched } = await upload(context, user, filePath, file, { meta, noTriggerImport, entity: previous });
+    const { upload: up, untouched } = await upload(context, user, filePath, file, { meta, noTriggerImport, entity: previous, file_markings });
```

Comment:
I don't see any change on '../database/file-storage' shouldn't marking be stored in elastic with indexFileToDocument call there ?

Reply:
@aHenryJard it's already taken care of in a previous PR ; it's stored in elastic properly